TESTS/spm/server/tests.c

```diff
@@ -128,7 +128,7 @@ PSA_TEST_SERVER(identity_during_call)
 
     signals = psa_wait_any(PSA_WAIT_BLOCK);
     if ((signals & TEST_MSK) == 0) {
-        test_status = ((test_status != PSA_SUCCESS) ? test_status : PSA_GENERIC_ERROR);
```

Comment:
I recall this coming in with a recent-ish PR. Was this not suppose to be the case?

Reply:
The removed condition is actually a dead code and generates Coverity warning.